src/app/components/top-menu/sequential-form/sequential-form.component.html

```diff
@@ -0,0 +1,44 @@
+<app-draggable-window [width]="'40%'" [height]="'65%'" [isResizeable]="true" [top]="'5%'" [right]="'2%'"
+                      (closeCallback)="this.close()" [title]="'Sequential Solver'">
+
+
+  <div [ngStyle]="{'visibility': this.showEdit ? 'visible' : 'hidden'}" class="matrix-input-content">
+    <textarea cols="40" rows="15" id="ef-text-input"
+                name="efTextInput" [(ngModel)]="this.efForm"></textarea>
+    <textarea cols="40" rows="15" id="ef-text-input2"
+                name="efTextInput" [(ngModel)]="this.solver.variableNames"></textarea>
+  </div>
+
+
+  <div></div>
```

Comment:
empty div and empty lines above should be removed.

Reply:
Agreed.